test/remote/gateways/remote_global_collect_test.rb

```diff
@@ -6,8 +6,12 @@ def setup
     @gateway_preprod = GlobalCollectGateway.new(fixtures(:global_collect_preprod))
     @gateway_preprod.options[:url_override] = 'preproduction'
 
+    @gateway_direct = GlobalCollectGateway.new(fixtures(:global_collect_direct))
```

Comment:
Since we are adding a new fixtures entry, we should stub `global_collect_direct` in the `fixtures.yml`.

Reply:
Thanks @naashton I will add the fixtures.